rct229/rulesets/ashrae9012019/section19/section19rule36.py

```diff
@@ -0,0 +1,94 @@
+from rct229.rule_engine.partial_rule_definition import PartialRuleDefinition
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.rulesets.ashrae9012019.data.schema_enums import schema_enums
+from rct229.schema.config import ureg
+from rct229.utils.jsonpath_utils import find_one
+from rct229.utils.pint_utils import CalcQ
+from rct229.utils.std_comparisons import std_equal
+
+ENERGY_RECOVERY = schema_enums["EnergyRecoveryOptions"]
+REQ_SENSIBLE_EFFECTIVENESS = 0.5
+REQ_LATENT_EFFECTIVENESS = 0.5
+
+
+class Section19Rule36(RuleDefinitionListIndexedBase):
+    """Rule 36 of ASHRAE 90.1-2019 Appendix G Section 19 (HVAC - General)"""
+
+    def __init__(self):
+        super(Section19Rule36, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section19Rule36.HVACRule(),
+            index_rmr="baseline",
+            id="19-36",
+            description="Baseline systems required to model energy recovery per G3.1.2.10 shall be modeled with a 50% enthalpy recovery ratio.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1-5(a)5 Building Envelope Modeling Requirements for the Proposed design",
+            is_primary_rule=False,
+            list_path="ruleset_model_descriptions[0].buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    class HVACRule(PartialRuleDefinition):
+        def __init__(self):
+            super(Section19Rule36.HVACRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+                required_fields={"$": ["fan_system"]},
+            )
+
+        def get_calc_vals(self, context, data=None):
+            fan_sys_b = context.baseline
+
+            sensible_eff_b = find_one(
+                "$.air_energy_recovery.design_sensible_effectiveness", fan_sys_b
+            )
+            latent_eff_b = find_one(
+                "$.air_energy_recovery.design_latent_effectiveness", fan_sys_b
+            )
+            ERV_OA_airflow_b = find_one(
+                "$.air_energy_recovery.outside_air_flow", fan_sys_b
+            )
+            ERV_EA_airflow_b = find_one(
+                "$.air_energy_recovery.exhaust_air_flow", fan_sys_b
+            )
+            hvac_min_oa_flow_b = find_one("$.minimum_outdoor_airflow", fan_sys_b)
+
+            return {
+                "sensible_eff_b": sensible_eff_b,
+                "latent_eff_b": latent_eff_b,
+                "ERV_OA_airflow_b": CalcQ("air_flow_rate", ERV_OA_airflow_b),
```

Comment:
What does CalcQ("air_flow_rate", ERV_OA_airflow_b) do?
```
                "ERV_OA_airflow_b": CalcQ("air_flow_rate", ERV_OA_airflow_b),
```

Reply:
The ```CalcQ``` function is used to specify units in the report. In other words, it isn't directly used in the rule check, but it's needed for the report module.

When a key (e.g., ```ERV_OA_airflow_b```) has a quantity (e.g., L/s), the key has to be specified with the ```CalcQ``` in the ```get_calc_vals``` method.